quil-rs/examples/generate_test_expressions.rs

```diff
@@ -68,7 +68,7 @@ fn parenthesized(expression: &Expression) -> String {
             parenthesized(right)
         ),
         Number(_) => format!("({})", expression.to_quil_or_debug()),
-        PiConstant => "pi".to_string(),
+        PiConstant() => "pi".to_string(),
```

Comment:
Idle thought: would it be worth applying `#[cfg(...)]` to this enum variant so that the Rust code follows convention as `PiConstant` and the PyO3 code is `PiConstant()` to support the `pyclass` macro?

I'm not sure that it _would_ be worth, but it would be a way to keep the Rust side conventional